package.json

```diff
@@ -30,22 +30,23 @@
     "lint": "eslint src"
   },
   "dependencies": {
-    "@babel/helper-module-imports": "7.0.0-beta.56",
+    "@babel/helper-module-imports": "7.0.0-rc.2",
     "rollup-pluginutils": "^2.3.0"
   },
   "peerDependencies": {
-    "@babel/core": "7 || ^7.0.0-beta"
+    "@babel/core": "7 || ^7.0.0-rc.2",
+    "rollup": "^0.60.0"
```

Comment:
this is for dynamic import support which rollup has unflagged since 0.60, it's probably possible to detect somehow also older versions (which had this support flagged under `experimental*` flag), but that would lead to some weirdo code and I thought it would be nice to push people for upgrading their rollups (and knowing from experience this is most often totally painless).